src/main/java/bio/terra/cli/businessobject/Server.java

```diff
@@ -37,6 +37,7 @@
   // (e.g. URLs, WSM single spend profile)
   private String samUri;
   private String workspaceManagerUri;
+  private String wsmDefaultSpendProfile;
```

Comment:
optional: consider renaming to `defaultSpendProfile`. It's kind of the default spend profile for the whole deployment. But it used only in WSM so far (and in our direct Sam group modifications)

Reply:
Since it's only used for WSM requests at the moment, I'm going to leave it as is. If this expands to other services (e.g. TDR) before we get a spend profile manager, then I agree with your suggestion and will update it here.